applications/FluidDynamicsApplication/fluid_dynamics_application.h

```diff
@@ -33,12 +33,14 @@
 #include "includes/define.h"
 #include "includes/kratos_application.h"
 
+#include "includes/deprecated_variables.h"
+
```

Comment:
Why are you including those here? I don't see them used on this file.

Reply:
greetings dear @jcotela ,

so far, I am almost done with the modifications, but only one thing please to make sure I got you correctly,
do you want me to include these below elements, and also to include their related documents:

#include "custom_elements/qs_vms.h"
#include "custom_elements/symbolic_navier_stokes.h"
#include "custom_elements/embedded_fluid_element.h"

Thanks and regards,

Reply:
Dear @elafnm: the elements you mention and the related code have been added to the fluid dynamics application while you were developing your code. Please do not remove them.

Reply:
thank you very much dear @jcotela for your prompt reply, 

sure will do,

just would like kindly to know also if I need to add these elements in my branch as well and compile it in my machine to resolve the conflict, b

because when I tried to add these three lines only and try to compile kratos, it complained that these elements are not existing, or you think no need for adding them in my branch for this pull request!

thanks again and my best regards,

Reply:
Dear @elafnm: since your changes to this file are very minor, I'd suggest just removing your modifications (the `#include "includes/deprecated_variables.h"` above). hopefully then the file can be merged without further conflicts.
If you ever find yourself in a similar situation, but want to keep the changes, I'd suggest merging the master to your branch (which will add all required files) and solving the conflict from there.